### PR TITLE
[10.0] migration of account_invoice_update_wizard

### PR DESCRIPTION
**Pull request by yvaucher**
This follows https://github.com/akretion/odoo-usability/pull/42

This is a almost straight forward port of the module from the version 9.0 done by @mdietrichc2c
I tried to change all available to change fields of a couple of invoice.

### CODE REVIEW

File: account_invoice_update_wizard/wizard/account_invoice_update.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
+# -*- coding: utf-8 -*-
+# © 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, _
+from odoo.exceptions import Warning as UserError
```
Comment (bealdav) [inline account_invoice_update_wizard/wizard/account_invoice_update.py]: No more Warning in v10

**Review by bealdav — APPROVED** [review]
ok except indicated

**Comment by fclementic2c** [comment]
@yvaucher Ok for me

However it is a shame that we cannot update also the analytical account and analytic tags. 
Is it possible do you think ?

**Comment by yvaucher** [comment]
@fclementic2c As it implies quite a lot of additions to that module I did the work in a second PR to not stop the current one: #68

**Comment by yvaucher** [comment]
@bealdav anything missing for a click on the merge button? :smile:

**Review by fclementic2c — APPROVED** [review]

**Comment by fclementic2c** [comment]
Can we have this module in the OCA account financial tools ?

**Pull request merged**